src/evict/evict_page.c

```diff
@@ -271,18 +273,19 @@ __evict_child_check(WT_SESSION_IMPL *session, WT_REF *parent)
  */
 static int
 __evict_review(
-    WT_SESSION_IMPL *session, WT_REF *ref, int exclusive, int *inmem_splitp)
+    WT_SESSION_IMPL *session,
+    WT_REF *ref,  int *inmem_splitp, uint32_t flags)
 {
 	WT_DECL_RET;
 	WT_PAGE *page;
 	WT_PAGE_MODIFY *mod;
-	uint32_t flags;
+	uint32_t reconcile_flags = WT_EVICTING;
```

Comment:
We don't do assignments in the declaration block. This should be split into two lines:

```
uint32_t reconcile_flags;

reconcile_flags = WT_EVICTING;
```